### PR TITLE
Zap API

### PR DESCRIPTION
**Pull request by binarymist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ If you do not wish to run NodeGoat on Heroku, please follow these steps to setup
 
 * Clone the github repository
 ```
-git clone https://github.com/OWASP/NodeGoat.git
+git clone https://github.com/binarymist/NodeGoat.git
```
Comment (ckarande) [inline README.md]: Possibly you didn't mean to include your fork' link this in PR?

Comment (binarymist) [inline README.md]: Oh, I just accepted a PR from my friend that is working with me in the workshop in two days time. I only merged this into my fork... I thought. Github must decide it needs to go upstream automatically. So no, it should only be in my fork.

**Comment by ckarande** [comment]
ok.. no problem..I will test it merge the PR later today.. thank you for the great work.

File: README.md (hunk 1 above)
Comment (ckarande) [inline README.md]: ok.. no problem..I will test it merge the PR later today.. thank you for the great work.

**Comment by ckarande** [comment]
@binarymist I had to remove host name from createServer() parameters to allow deployment on Heroku. I think this host name is not mandatory for testing via ZAP API. Please let me know if otherwise and any alternatives.

**Pull request merged**